perfect_divide.py

```diff
@@ -0,0 +1,17 @@
+"""
+ generator that find a perfect number
+"""
+def pefect_num():
+    n=2
+    while n:
+        sum1 = 0
+        l = [i for i in range(1, n) if(n % i == 0)]
```

Comment:
The usage of list comprehension is great, but again, DO NOT use single letter names for your variables.